src/app/Main.java

```diff
@@ -109,15 +119,14 @@ private static User getUser(Scanner sn) {
 
     private static Book getBook(Scanner sn) {
```

Comment:
Can you rename this to `getNewBook()` or `createNewBook()` ?

Reply:
Okey, no problem.